### PR TITLE
GDB-6401 Add cluster settings panel

### PR DESCRIPTION
**Pull request by yordanalexandrov**
- added a button to show a panel with cluster settings

### CODE REVIEW

File: src/pages/cluster-management/clusterInfo.html [hunk 1]
```diff
@@ -18,139 +20,92 @@ <h1>
      size="75">
 </div>
 
-<div ng-hide="loader" class="clearfix mb-2">
-    <button type="button" class="btn btn-secondary delete-cluster-btn mr-2" ng-if="clusterConfiguration"
+<div ng-hide="loader" class="clearfix mb-2" ng-if="clusterConfiguration">
```
Comment (svilenvelikov) [inline src/pages/cluster-management/clusterInfo.html:23]: Using ng-if and ng-hide on same node is confusing

File: src/pages/cluster-management/clusterInfo.html [hunk 2]
```diff
@@ -18,139 +20,92 @@ <h1>
...
+    <div class="rdf-side-panel-content break-word-alt p-1 pt-2">
+       <span class="toggle-sidepanel" title="{{'visual.collapse.sidepanel' | translate}}" ng-click="toggleSidePanel()">
+            <em ng-class="shouldShowClusterSettingsPanel ? 'icon-caret-right' : 'icon-caret-left'"></em>
+        </span>
+        <button class="close mb-1 ml-1" ng-click="toggleSidePanel()"></button>
+        <h3 class="hovered-parent">
+            {{'cluster_management.cluster_configuration.label' | translate}}
+        </h3>
+        <h4>{{'cluster_management.cluster_configuration.configuration_label' | translate}}</h4>
+        <div class="rdf-l2ist">
```
Comment (svilenvelikov) [inline src/pages/cluster-management/clusterInfo.html:57]: `rdf-l2ist`?

File: src/pages/cluster-management/clusterInfo.html (hunk 1 above)
Comment (yordanalexandrov) [reply src/pages/cluster-management/clusterInfo.html:23]: Ah.. yes it is. I'm just not used to having a few ways of showing/hiding stuff. Removed the `ng-hide` directive.

File: src/pages/cluster-management/clusterInfo.html (hunk 2 above)
Comment (yordanalexandrov) [reply src/pages/cluster-management/clusterInfo.html:57]: This has to be removed. Removed the whole `div` wrapper.

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**